### PR TITLE
chore: Minor code refinements.

### PR DESCRIPTION
**Pull request by haruhi55**
- Improve thread arrangement within a thread block for better efficiency.
- Minor code refinements.

### CODE REVIEW

File: include/cell/traits/b2b_gemm.hpp [hunk 1]
```diff
@@ -9,7 +9,7 @@ using namespace cute;
 namespace tl = tile_layout;
 
 template <typename Element_, typename CtaTileShape,
-          typename WarpShape = tiledcuda::cell::TileShape<1, 1>,
+          typename WarpShape = tiledcuda::cell::TileShape<4, 1>,
```
Comment (KuangjuX) [inline include/cell/traits/b2b_gemm.hpp:12]: Add a comment for flash-attention-v2?

File: include/cell/traits/bmm.hpp [hunk 2]
```diff
@@ -10,7 +10,7 @@ using namespace cute;
 namespace tl = tile_layout;
 
 template <typename Element_, typename CtaTileShape,
-          typename WarpArrangement = tiledcuda::cell::TileShape<1, 2>,
+          typename WarpArrangement = tiledcuda::cell::TileShape<2, 2>,
```
Comment (KuangjuX) [inline include/cell/traits/bmm.hpp:13]: I’m quite curious about how the default configuration`typename WarpArrangement = tiledcuda::cell::TileShape<2, 2>` was considered?

**Review by KuangjuX — COMMENTED** [review]
LGTM overall.

File: include/cell/traits/b2b_gemm.hpp (hunk 1 above)
Comment (haruhi55) [reply include/cell/traits/b2b_gemm.hpp:12]: but I think fused two dependent gemms is much general than flash attention.

File: include/cell/traits/bmm.hpp (hunk 2 above)
Comment (haruhi55) [reply include/cell/traits/bmm.hpp:13]: It is not necessary to use small threads setting.

File: include/cell/traits/b2b_gemm.hpp (hunk 1 above)
Comment (KuangjuX) [reply include/cell/traits/b2b_gemm.hpp:12]: Ok, I got it.

**Review by KuangjuX — APPROVED** [review]
LGTM.

**Pull request merged**